cmd/agent/dist/conf.d/snmp.d/auto_conf.yaml

```diff
@@ -58,7 +58,7 @@ instances:
 
     ## @param privProtocol - string - optional
     ## Privacy type to use when connecting to your SNMP devices.
-    ## It can be one of: DES, 3DES, AES, AES192, AES256, AES192C, AES256C.
```

Comment:
Should we say that `AES` == `AES128` ? I've seen this question come up a few times

Reply:
@FlorianVeaux Good idea. 

Added this https://github.com/DataDog/datadog-agent/pull/10233/commits/dd37c0831a79357d67f8f056db1ce88d227d1082